build/dependencies.props

```diff
@@ -139,7 +148,6 @@
     <MicrosoftApplicationInsightsAspNetCorePackageVersion>2.1.1</MicrosoftApplicationInsightsAspNetCorePackageVersion>
     <MicrosoftAspNetIdentityEntityFrameworkPackageVersion>2.2.1</MicrosoftAspNetIdentityEntityFrameworkPackageVersion>
     <MicrosoftAspNetWebApiClientPackageVersion>5.2.6</MicrosoftAspNetWebApiClientPackageVersion>
-    <MicrosoftAzureDocumentDBCorePackageVersion>1.7.1</MicrosoftAzureDocumentDBCorePackageVersion>
```

Comment:
While this package versions are going away from here, how would EF Core control package versions in future?

Reply:
However you want :) Think of EFCore as a sovereign repo now. You are welcome to keep using the KoreBuild conventions like build/dependencies.props, or you can do something else.

This also means aspnetci won't push changes to the master branch with dependency updates. The recommended replacement for this is the .NET Core 3.0 dependency flow tools. See https://github.com/dotnet/arcade/blob/master/Documentation/DependenciesFlowPlan.md

Reply:
I highly urge most repos to stick to the common patterns, though.